src/containers/ResetPassword/ResetPassword.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ResetPassword from "./ResetPassword";
+import { shallow, mount } from "enzyme";
+import { TextField } from "@material-ui/core";
+
+it("should render ResetPassword correctly", () => {
+  const wrapper = shallow(<ResetPassword />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it("should render TextFiels correctly", () => {
```

Comment:
One snapshot for the whole component is enough

Reply:
I really do it, so when I interact with api, I need to spy on the both functions of button and form, so shallow the TextField gonna save much headache 😁